src/migrations/types/callout-state.ts

```diff
@@ -0,0 +1,5 @@
+export enum CalloutState {
```

Comment:
Please move this state as part of the migration using it. The state is a snapshot in time and if the states change, you change this enum, and then re-running the migrations will change its original behavior, which is not intended.